Commands/Commands/uservoice.js

```diff
@@ -225,7 +225,7 @@ commands.comment = {
       })
     }).catch(e => {
       if (e === 'Not found') {
-        msg.reply("I was unable to find your details, make sure you've logged into the website at least once.").then(errmsg => {
+        msg.reply("I was unable to find your details, make sure you've logged into the website at <https://`+ config.uservoice.subdomain + `.` + config.uservoice.domain + `> at least once.").then(errmsg => {
```

Comment:
@LW001 You tried to use template syntax outside of template literal strings, Travis is therefore correct in erroring out.
Use the spoon-feeding @ThaTiemsz gave you.